diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5c08246c..29b9dcab 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -126,7 +126,7 @@ endif()
 
 # Build all test.  The test are assumed to reside inside of the directory 
 # "test" within each module.  The main
-build_test()
+#build_test()
 
 # The EventDisplay is stand-a-lone and _needs_ to be after everything else
 # so that it can clear the cache variables and construct its own dictionary.
Submodule cmake contains modified content
diff --git a/cmake/FindONNXRuntime.cmake b/cmake/FindONNXRuntime.cmake
index 348e349..df03c19 100644
--- a/cmake/FindONNXRuntime.cmake
+++ b/cmake/FindONNXRuntime.cmake
@@ -47,16 +47,16 @@ macro(download_onnxruntime version destination)
   endif()
 
   # Extract the archive
-  execute_process(COMMAND tar -zxvf ${file_prefix}.tgz
+  execute_process(COMMAND tar -zxvf ${file_prefix}.tgz --strip-components=1
                   WORKING_DIRECTORY ${destination})
 
   # Set the path to the include and libraries to facilitate finding the
   # dependency.
   set(ONNXRuntime_INCLUDE_DIRS
-      "${destination}/${file_prefix}/include"
+      "${destination}/include"
       CACHE INTERNAL "")
   set(ONNXRuntime_LIBRARIES
-      "${destination}/${file_prefix}/lib"
+      "${destination}/lib"
       CACHE INTERNAL "")
   set(ONNXRuntime_VERSION
       ${version}
@@ -65,15 +65,15 @@ macro(download_onnxruntime version destination)
 endmacro()
 
 if(NOT TARGET ONNXRuntime::Interface)
-  find_path(
-    onnxruntime_include_dir
-    NAMES onnxruntime_cxx_api.h
-    PATHS ${ONNXRuntime_INCLUDE_DIRS}
-    PATH_SUFFIXES onnxruntime)
+#  find_path(
+#    onnxruntime_include_dir
+#    NAMES onnxruntime_cxx_api.h
+#    PATHS ${ONNXRuntime_INCLUDE_DIRS}
+#    PATH_SUFFIXES onnxruntime)
 
   # If onnxruntime isn't found, download it.
   if(NOT onnxruntime_include_dir)
-    download_onnxruntime(1.2.0 ${CMAKE_INSTALL_PREFIX}/external)
+    download_onnxruntime(1.2.0 ${CMAKE_INSTALL_PREFIX})
   endif()
 
   # Create the target
